### ISSUE TITLE
Update NormedPerson to serialization logic similar to NormedPublication

### ISSUE THREAD

**Issue by rickjohnson** [share-research/pace-admin#137]
Similar to NormedPublication, do the following
- [ ] Create JSON mapping file for person similar to normedPublicationObjectToCSVMap.json
- [ ] Convert loadPerson to static method in NormedPerson.ts and rename to loadFromCSV so that it would be called in the form NormedPerson.loadFromCSV with returning NormedPerson[]. Use JSON mapping file in load
- [ ] Create writeToCSV static method in NormedPerson.ts takes NormedPerson[] and filepath as parameters, using JSON mapping
- [ ] Convert NormedPerson from and a Typescript interface to a Typescript class similar to NormedPublication
- [ ] Convert usages of loadPerson in other scripts to use new method
- [ ] Move queryNormalizedPeople methods into NormedPerson.ts (perhaps rename to loadFromDB ?)
- [ ] Audit the db attribute names in DB and NormedPerson to see if they can be normalized together

**Comment by jeremyf** [comment]
@rickjohnson I have a question about the purpose of this work; In the work breakdown, it looks like I'm gathering up methods that are presently scattered throughout the code base.  The goal being to put those methods in a common class.

What confuses me is the purpose of the "Create JSON mapping file for person similar to normedPublicationObjectToCSVMap.json" task.  In the [`normedPublicationObjectToCSVMap.json`](https://github.com/share-research/pace-admin/blob/5b2b1bc15e151f2970a0bd04a23c674f8bf96233/ingest/modules/normedPublicationObjectToCSVMap.json) this looks like the JSON keys are the attribute names for the interface/ingest class.  And the values are the CSV column name.

I guess I'm struggling to see the necessity of this mapping file.  Do we already have a case where this process would have two different structured CSVs that would load the normedPerson?  It seems that this is an unnecessary optimization at this point (e.g. we're encapsulating the read/write from CSV).  In reading the code, it's create a bit of indirection to comprehension. (e.g. to know what's happening, I need three files open: CSV, JSON Map, and Code)

**Comment by rickjohnson** [comment]
@jeremyf you are right it may not be necessary.  I originally created it for pubs thinking it might be good to have these mappings be configurable and originally had it in the config dir.  I then moved it 'closer' to the normedPublication code after realizing a change to it might be a code-breaking change.  So...  an improvement could be to fold that into the normedPublication class, and have something similar embedded in the normedPerson class

**Comment by jeremyf** [comment]
That's kind of been my framing; I'll see where I go with the implementation.

It's been difficult scheduling time because I have a meeting, then an hour then a meeting.  I'm going to block out time later this week.

**Comment by jeremyf** [comment]
I've updated the checkboxes to reflect PR#141.

However, I'm at a loss with several next steps regarding this ticket.  In particular I am not finding any `queryNormalizedPeople` in the code-base; So I'm hard-pressed to go down that path.

Is this something you have on your machine or in another branch?
